browser/react/components/Joey.js

```diff
@@ -14,9 +15,14 @@ export default class Joey extends React.Component {
   render () {
     return (
       <a-entity id="room" position="0 0 0">
+        <Room floorWidth="50"
+              floorHeight="50"
+              wallHeight="25"
+              wallColor="blue"
+              floorColor=""
```

Comment:
part of me thinks that we should have a woodish brown color for "progressive enhancement" purposes. I don't think this needs to be done before merging. Just informational.